udata/harvest/actions.py

```diff
@@ -28,12 +28,16 @@ def list_backends():
     return backends.get_all().values()
 
 
-def list_sources(owner=None):
+def list_sources(owner=None, page=None, page_size=None):
     '''List all harvest sources'''
     sources = HarvestSource.objects.visible()
     if owner:
-        return list(sources.owned_by(owner))
-    return list(sources)
+        sources = sources.owned_by(owner)
+    if page_size:
+        page = max(page or 1, 1)
+        return sources.paginate(page, page_size)
```

Comment:
No misunderstanding: the returned value in case of pagination is not a list but an object like this:
```
{items: [], total: 100, page: 2, page_size: 10}
```
I chose to have only one function with different return depending on pagination active or not but I can split in 2 functions to always have the same return format (with forced pagination in one).